.code-samples.meilisearch.yaml

```diff
@@ -1187,3 +1188,14 @@ index_settings_tutorial_api_put_setting_1: |-
 index_settings_tutorial_api_task_1: |-
   curl \
     -X GET 'http://localhost:7700/tasks/TASK_UID'
+get_search_cut_off_1: |-
```

Comment:
I was thinking, maybe `get_search_cutoff_1` is more consitent with the naming?

Reply:
hmmmm…you lead me down a spelling rabbit hole. 

I was using `cut-off`, as suggested by the Oxford English Dictionary (aka OED). According to that spelling, `get_search_cut_off` would be more accurate.

However OED is heavily skewed towards British English. The Merriam-Webster and the New Oxford dictionaries both lean American and prefer `cutoff`. Since our documentation is written in American English, I suppose it's better to follow that spelling, even if both are valid.

So, in short, yes, you are correct.

Reply:
I was saying "consistent" based on OUR naming 😊 which is `searchCutoffMs` and not `SearchCutOffMs`

Reply:
That's also true. And, I guess, another reason to use the preferred American spelling.